Section 3.md

```diff
@@ -99,6 +105,13 @@ While it does not yet possess technologies like Gravitational Singularity reacto
 
 While usually the larger corporations devise their own power grid and manage their own planets, moons, orstations, FPE is often consulted and hired when a less than usual approach needs to be taken, and their products are on high demand.
 
+- **Morpheus Cyberkinetics**
+
+Originally a small company focused on research of the interface of the human mind to cybernetic parts, founded just prior to the explosion in robotics technology. This company produced the first of the truly high-end cybernetics, interfacing directly with the human nervous system, even giving a very rudimentary sense of touch. There are some rumors that Zeng-Hu had seized their results through corporate espionage, quickly producing their own line of prosthetics that eclipsed Morpheus� own.
+
+Morpheus, however, had one breakthrough that is entirely attributed to it, the �MMI�. This device not only allowed for people to be �revived� by having their brains transplanted into it, it also allowed them to access and interface with many custom-built robots and other devices.(However, at the time, it was most common to find cyborgs in modified drone chassis, as a proper chassis was simply too expensive for many.)
```

Comment:
"Later in years, Morpheus came to yet another break through, this time fully attributed to them, with the MMI. The MMI not only allowed for people to be revived through brain transplants, but also allowed for them to access and interface with custom-built machinery, such as cyborg equipment. However, at the time, it was far more common to find MMIs in modified drone chassis, as the proper cyborg chassis were too expensive for most."